package.json

```diff
@@ -65,6 +65,7 @@
     "broccoli-merge-trees": "^1.1.0",
     "broccoli-string-replace": "^0.1.1",
     "chalk": "^1.1.0",
+    "electron-forge": "isleofcode/electron-forge",
```

Comment:
TODO: revert to mainline electron-forge once https://github.com/electron-userland/electron-forge/pull/113 merges